Source/Scene/Cesium3DTilesetTraversal.js

```diff
@@ -316,6 +316,13 @@ define([
             return;
         }
 
+        if (tile.hasTilesetContent && tile.children.length > 0) {
```

Comment:
I'd add a quick comment to explain why this is needed.

Reply:
Also we're already defining `var hasChildren = tile.children.length > 0;` on line 334, move it up here?